operating/manage-user-access/understanding-multi-tenancy.adoc

```diff
@@ -0,0 +1,97 @@
+* xref:../../operating/manage-user-access/manage-role-based-access-control-3630.adoc#create-a-custom-role-3630_manage-role-based-access-control[Create a custom role]
+
+[id="understanding-resource-scoping"]
+== Understanding resource scoping
+
+{product-title-short} includes resources which are used within RBAC.
+In addition to associating permissions for a resource, each resource is also scoped.
+
+In {product-title-short}, resources are scoped as:
+
+* Global scope, where a resource is not assigned to any cluster or namespace.
+* Cluster scope, where a resource is assigned to particular cluster(s).
+* Namespace scope, where a resource is assigned to particular namespace(s).
+
+The scope of resources is important when creating custom access scopes.
+Custom access scopes are used to create multi-tenancy within {product-title-short}.
+
+Only resources which are cluster or namespace scoped are applicable for scoping in access scopes. Globally scoped resources are not scoped by access scopes.
+
+Thus, multi-tenancy within {product-title-short} can only be achieved for resources that are scoped either by cluster or namespace.
+
+[id="multi-tenancy-for-namespace"]
+== Multi-tenancy per namespace
+
+One of the most common examples for multi-tenancy within {product-title-short} is associating users with a specific namespaces and only allowing them access to their specific namespace.
+
+Below is an example of combining a custom permission set, access scope, and role. The user or group assigned with this role will only be able to see CVE information, violations, and information about deployments in the particular namespace or cluster scoped to them.
+
+.Procedure
+
+. In the {product-title-short} portal, select *Platform Configuration* -> *Access Control*.
+. Select *Permission Sets*.
+. Click *Create permission set*.
+. Enter a *Name* and *Description* for the permission set.
+. Select the following resources and access level and click *Save*:
+** `READ` Alert
+** `READ` Deployment
+** `READ` DeploymentExtension
+** `READ` Image
+** `READ` K8sRole
+** `READ` K8sRoleBinding
+** `READ` K8sSubject
+** `READ` NetworkGraph
+** `READ` NetworkPolicy
+** `READ` Secret
+** `READ` ServiceAccount
+. Select *Access Scopes*.
+. Click *Create access scope*.
+. Enter a *Name* and *Description* for the access scope.
+. In the *Allowed resources* select the namespace you want to use for scoping and click *Save*.
+. Select *Roles*.
+. Click *Create role*.
+. Enter a *Name* and *Description* for the role.
+. Select the previously created *Permission Set* and *Access scope* for the role and click *Save*.
+. Assign the role to your desired user or group following xref:../../operating/manage-user-access/manage-role-based-access-control-3630.adoc#assign-role-to-user-or-group_manage-role-based-access-control[these instructions].
```

Comment:
```suggestion
. Assign the role to your required user or group. See xref:../../operating/manage-user-access/manage-role-based-access-control-3630.adoc#assign-role-to-user-or-group_manage-role-based-access-control[Assigning a role to a user or a group].
```
[IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=word-usage) disallows use of "desired"
Also we have to use the title of the section or doc in the links